### ISSUE TITLE
Bug: New Update Causes Gluetun to Fail Using Wireguard With Custom VPNSP: VPN Provider Name Not Provided

### ISSUE THREAD

**Issue by gist901** [qdm12/gluetun#822]
### Is this urgent?

Yes

### Host OS

Debian Buster

### CPU arch

armv7l

### VPN service provider

TorGuard

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Latest

### What's the problem 🤔

After updating my Gluetun container to `latest`, it fails to connect to my VPN provider via Wireguard. If I update my compose file with the tag `v3.27.0` and reload the Gluetun container connects successfully as usual.

### Share your logs

```log
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version latest built on 2022-01-26T22:45:19.321Z (commit 1e3f878)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2022/01/26 21:57:20 INFO storage: merging by most recent 11100 hardcoded servers and 11100 servers read from /gluetun/servers.json
2022/01/26 21:57:20 ERROR failed validating updater settings: VPN provider name is not valid: custom at index 0
2022/01/26 21:57:20 INFO Shutdown successful
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version latest built on 2022-01-26T22:45:19.321Z (commit 1e3f878)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2022/01/26 21:57:23 INFO storage: merging by most recent 11100 hardcoded servers and 11100 servers read from /gluetun/servers.json
2022/01/26 21:57:23 ERROR failed validating updater settings: VPN provider name is not valid: custom at index 0
2022/01/26 21:57:23 INFO Shutdown successful
```

### Share your configuration

```yml
version: '3.8'

services:

  gluetun:
    container_name: gluetun
    image: qmcgaw/gluetun:latest
    hostname: gluetun
    cap_add:
      - NET_ADMIN
    ports:
      - 8112:8112/tcp
    volumes:
      - /mnt/travel/dockerConfig/gluetun:/gluetun
    restart: unless-stopped
    environment:
      - PUID=1001
      - PGID=1001
      - TZ=America/<city>
      - VPNSP=custom
      #OpenVPN:
      #- VPN_TYPE=openvpn
      #- OPENVPN_CUSTOM_CONFIG=/gluetun/custom.conf
      #- OPENVPN_USER=
      #- OPENVPN_PASSWORD=
      #- FIREWALL_VPN_INPUT_PORTS=17383
      #Wireguard:
      - VPN_TYPE=wireguard
      - WIREGUARD_ENDPOINT_IP=<-VPN IP->
      - WIREGUARD_ENDPOINT_PORT=1443
      - WIREGUARD_PUBLIC_KEY=<key>
      - WIREGUARD_PRIVATE_KEY=<key>
      - WIREGUARD_ADDRESS=10.13.128.85/32
      - FIREWALL_VPN_INPUT_PORTS=17383

networks:
  default:
    external: true
    name: br0
```

**Comment by VictorDrijkoningen** [comment]
I have the same issue, except my error states " 2022/01/27 11:53:47 ERROR failed validating updater settings: VPN provider name is not valid: custom at index 0"

I also have custom vpn like the wiki config [wiki link ](https://github.com/qdm12/gluetun/wiki/Custom-provider)

**Comment by joegreene** [comment]
For what it's worth to note I'm receiving the same exact error as @gist901 except with a custom OpenVPN config (provided by Windscribe VPN).

**Comment by qdm12** [comment]
Fixed by 9fb085f3613ddb033dc85ef106ef46f67c1efa05 😉 Thanks everyone for taking the time to report the problem!
